README.md

```diff
@@ -54,3 +63,5 @@ If you have fixed an issue or added a functionality, please feel free to submit
 
 ### Build
 Please see the [wiki](https://github.com/bird-sanctuary/bluejay/wiki/Building-from-source) for instructions on how to build Bluejay from source.
+
+TEST
```

Comment:
Has this text inserted unadvertedly?

Reply:
Yes, this was just to trigger the build and make sure the commenter works ;-) forgot to revert it...